<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DMXClass</class>
 <widget class="QMainWindow" name="DMXClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DMX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="state">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSlider" name="Slider12">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label12">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>260</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider11">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label11">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider10">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label10">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label9">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider7">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label8">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label7">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider8">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider6">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label5">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="Slider3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QSlider" name="Slider4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>19</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Label4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Label1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DMX.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Slider1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>569</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider11</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider12</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DMXClass</receiver>
   <slot>Update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DMXClass</receiver>
   <slot>ResetDMX1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DMXClass</receiver>
   <slot>ResetDMX2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>DMXClass</receiver>
   <slot>ResetDMX3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>590</x>
     <y>344</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>DMXClass</receiver>
   <slot>Auto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Update()</slot>
  <slot>ResetDMX1()</slot>
  <slot>ResetDMX2()</slot>
  <slot>ResetDMX3()</slot>
  <slot>Random()</slot>
  <slot>Auto()</slot>
 </slots>
</ui>
